utils/build/docker/php/common/php.ini

```diff
@@ -25,3 +25,8 @@ ddappsec.helper_socket_path=/tmp/ddappsec.sock
 ddappsec.helper_lock_path=/tmp/ddappsec.lock
 ddappsec.helper_extra_args=--log_level debug
 ddappsec.helper_log_file=/tmp/helper.log
+
+datadog.appsec.enabled_on_cli=0
+datadog.trace.agent_port=8126
```

Comment:
On php, default agent port is `0` and on system tests it is expected to be `8126`